src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/TestHtmlDocumentPublisher.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+
+namespace Microsoft.VisualStudio.Razor.LanguageClient.Cohost;
+
+internal sealed class TestHtmlDocumentPublisher : IHtmlDocumentPublisher
+{
+    private readonly List<(TextDocument, string, ChecksumWrapper)> _publishes = [];
+
+    public List<(TextDocument Document, string Text, ChecksumWrapper Checksum)> Publishes => _publishes;
```

Comment:
>public List<(TextDocument Document, string Text, ChecksumWrapper Checksum)> Publishes => _publishes; [](http://example.com/codeflow?start=4&length=100)

nit: Not sure if I prefer this instead:

```
public List<(TextDocument Document, string Text, ChecksumWrapper Checksum)> Publishes { get; } = [];
```

Reply:
I've always just personally preferred making my own backing fields. Maybe that will change as `field` becomes more prevalent 🤷‍♂️ Not really fussed either way for this little test class, I didn't even read it, I just use the "Move to foo.cs" code fix :P